boards/arm/nrf52_sparkfun/nrf52_sparkfun_defconfig

```diff
@@ -0,0 +1,21 @@
+CONFIG_ARM=y
+CONFIG_SOC_FAMILY_NRF5=y
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52832_QFAA=y
+CONFIG_BOARD_NRF52_BLENANO2=y
+
+# Enable MPU
+CONFIG_ARM_MPU_NRF52X=y
+
+# enable uart driver
+CONFIG_SERIAL=y
+CONFIG_UART_NRF5=y
+CONFIG_UART_NRF5_FLOW_CONTROL=n
+
+# enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# bluetooth
+CONFIG_BT=y
+CONFIG_BT_CTLR=y
```

Comment:
I saw that reste pin is conected to button at this board - meaybe it would be good to add pinreset support by default:
```
# additional board options
CONFIG_GPIO_AS_PINRESET=y
```